benchmarking/daint_single_node/stdout_to_json.py

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env python3
+
+# This script parses the standard output of a FV3GFS run to extract timing
+# information and convert it to the JSON format which is used by the performance
+# monitoring script.
+
+# 2021/01/22 Oliver Fuhrer, Vulcan Inc, oliverf@vulcan.com
+
+import os, re, click, glob, time, datetime, json, sys
+
+# this dict maps the names of the timer in the JSON file to the actual
+# timers reported in stdout of FV3GFS, multiple entries are accumulated
+TIMER_MAPPING = {
+  "total": ["Total runtime"],
+  "init": ["1-Initialization", "2-Main-loop-1st-trip"],
+  "main_loop": ["3-Main-loop"],
+  "fv_dynamics": ["3.1.1-fv_dynamics"],
+  "dyn_core": ["3.1.1.1-dyn_core"],
+  "tracer_adv": ["3.1.1.2-Tracer-advection"],
+  "remapping": ["3.1.1.3-Remapping"]
+}
+
+def num(s):
+    try:
+        return int(s)
+    except ValueError:
+        return float(s)
+
+
+@click.command()
+@click.option('--stdout_file_regex', default='slurm-*.out', help='regex pattern to identify stdout file')
+@click.argument('run_directory', nargs=1)
+def stdout_to_json(stdout_file_regex, run_directory):
```

Comment:
```suggestion
def stdout_to_json(stdout_file_regex, run_directory):
"""
This function parses the standard output of a FV3GFS run in the run_directory to extract timing information and convert it to the JSON format.
"""
```

or a short docstring along those lines?